ui/app/components/search-select.js

```diff
@@ -0,0 +1,99 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  'data-test-component': 'search-select',
+  classNames: ['field', 'search-select'],
+  store: service(),
+  /*
+   * @public
+   * @param Function
+   *
+   * Function called when any of the inputs change
+   * accepts a single param `value` that is the
+   * result of calling `toVal()`.
+   *
+   */
+  onChange: () => {},
+
+  /*
+   * @public
+   * @param String | Array
+   * A comma-separated string or an array of strings.
+   * Defaults to an empty array.
+   *
+   */
+  inputValue: computed(function() {
+    return [];
+  }),
+  selectedOption: null,
+  selectedOptions: null,
+  options: null,
+  shouldUseFallback: false,
+  init() {
+    this._super(...arguments);
+    this.set('selectedOptions', this.inputValue || []);
+  },
+  fetchOptions: task(function*() {
+    for (let modelType of this.models) {
```

Comment:
is `models` an arg to the component? Should we default it to `[]` like inputValue so this doesn't error if you don't pass it in?